### PR TITLE
Added support for sellercentral.amazon.com

### PR DESCRIPTION
**Pull request by davext**
Amazon was supported, but Amazon's seller central was not. Also remove img tags because it was reversing profile pictures

### CODE REVIEW

File: src/config/inversion-fixes.config [hunk 1]
```diff
@@ -1,7 +1,6 @@
 *
 
 INVERT
-img
```
Comment (Gusted) [inline src/config/inversion-fixes.config:4]: This should be a separate PR, but would break a lot of current setups/configurations

**Review by davext — COMMENTED** [review]
Adding img back

File: src/config/inversion-fixes.config [hunk 2]
```diff
@@ -1966,6 +1966,24 @@ INVERT
...
+}
+.small,.account-switch-icon{
+   fill: black !important;
+}
+.locale-icon-wrapper{
+    color:black;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
+}
+.logo-container{
+filter: brightness(0%);
+}
```
Comment (Gusted) [inline src/config/inversion-fixes.config:1983]:
```suggestion
.locale-icon-wrapper{
    color: black !important;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
}
.logo-container {
    filter: brightness(0%) !important;
}
```

**Review by Gusted — APPROVED** [review]
